docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +29,18 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To use Iceberg, you need:
+
+* Iceberg table format compatible with Apache Iceberg format version 1.
```

Comment:
Lets move this into the introduction as discussed with @electrum

Reply:
Changing to read something like

> The Iceberg connector supports Apache Iceberg format version 1